### DISCUSSION TITLE
工作空间 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1060] [Announcements]
# 工作空间 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/instance/workspace/

**Comment by liyuerich** [comment]
简单来说，当一个多云实例或多云命名空间之后

是不是缺少了动词 “被绑定”？

**Comment by windsonsea** [comment]
Fixed
